readme.md

```diff
@@ -81,7 +83,8 @@ var Panda = function(name, age) {
 ```
 
 Your Answer:
-```text
+```var eat_bamboo = function(){
+      this.num_bamboo_eaten += [i]
 
 ```
```

Comment:
This `eat_bamboo` function is not attached to Panda. In order for it to be accessible by all pandas, it needs to included in the Panda prototype like so...

```js
Panda.prototype.eat_bamboo = function() {
  this.num_bamboo_eaten = this.num_bamboo_eaten + 1;
}
```